lib/Form/Type/FieldType/MapLocationFieldType.php

```diff
@@ -42,36 +44,41 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'latitude',
                 NumberType::class,
                 [
-                    'label' => 'content.field_type.ezgmaplocation.latitude',
+                    'label' => /** @Desc("Latitude") */ 'content.field_type.ezgmaplocation.latitude',
                     'required' => $options['required'],
                     'attr' => [
                         'min' => -90,
                         'max' => 90,
-                        'step' => 'any',
+                        'step' => 0.000001,
```

Comment:
Is this option required at all? I don't see a point in defining steps for lat/long numbers as they're very precise and changing them by decreasing/increasing has no value.

Reply:
We want to use native HTML5 inputs, in this case it's `number` input type. User is still allowed to provide value manually.

Reply:
If it's required from HTML5 standpoint, then no objections.